docs/structure-of-a-contract.rst

```diff
@@ -43,19 +43,26 @@ visibility.
 Functions
 =========
 
-Functions are the executable units of code within a contract.
+Functions are the executable units of code. Functions are usually
+defined inside a contract, but they can also be defined outside of
+contracts.
 
 ::
 
     // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >=0.4.0 <0.8.0;
+    pragma solidity >=0.7.1 <0.8.0;
```

Comment:
```suggestion
    pragma solidity >0.7.0 <0.8.0;
```

The command line tests are failing because of this, right? Or maybe because of something else.